frog/imports/api/graphs.js

```diff
@@ -159,5 +219,17 @@ Meteor.methods({
       const conid = connections.map(x => x._id);
       Connections.remove({ _id: { $nin: conid }, graphId });
     }
+  },
+  'graph.flush.all': graphId => {
+    Graphs.remove(graphId);
+    Activities.remove({ graphId });
+    Operators.remove({ graphId });
+    Connections.remove({ graphId });
+    Sessions.remove({ graphId });
+  },
+  'graph.flush.activity': activityId => {
+    Operators.remove({ from: activityId });
```

Comment:
I don't know, I really only moved this function as it made more sense for it to be in api/graphs instead of api/activities, but I did not change it's definition